File: src/main/java/guess/Game.java
package guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret) {
            System.out.print("Plz enter number:");
            number = scanner.nextInt();
            if (number < secret) {
                System.out.println("Higher");
            } else if (number > secret) {
                System.out.println("Lower");
            } else
                System.out.println("Currect");
        }

/*        int i = 0;
        do {
            System.out.println(i);
            i++;

        } while (i < 3);*/

    }

}

File: src/main/kotlin/guess/Game.kt
package guess

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10) + 1
    println(secret)
//    val scanner = Scanner(System.`in`)

    var number = 0
    while (number != secret) {
        print("Plz enter a number:")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        if (number < secret) {
            println("Higher")
        } else if (number > secret) {
            println("Lower")
        } else {
            println("Great number is $secret")
        }
    }

    var i = 0;
    do {
        System.out.println(i);
        i++;
    } while (i != 10);
}